.circleci/config.yml

```
@@ -0,0 +1,145 @@
+version: 2.1
+
+orbs:
+  aws-cli: circleci/aws-cli@0.1.13
+
+aliases:
+  - &workdir /home/circleci/build
+
+  - &container_config
+    docker:
+      - image: circleci/node:10
+    working_directory: *workdir
+
+  - &checkout checkout
+
+  - &install_aws_cli aws-cli/install
+
+  - &configure_aws_cli
+    aws-cli/configure:
+      aws-region: 'AWS_REGION'
+      aws-access-key-id: 'AWS_ACCESS_KEY_ID'
+      aws-secret-access-key: 'AWS_SECRET_ACCESS_KEY'
+
+  - &save_cache
+    save_cache:
+      paths:
+        - active-chargers-consumer/node_modules
+        - active-chargers-sweep/node_modules
+      key: dependencies-cache-{{ checksum "yarn.lock" }}
+
+  - &restore_cache
+    restore_cache:
+      keys:
+        - dependencies-cache-{{ checksum "yarn.lock" }}
+        # fallback to using the latest cache if no exact match is found
+        - dependencies-cache
+
+  - &install_dependencies
+    run:
+      name: Install Dependencies
+      command: yarn install-all --frozen-lockfile
```

`yarn install-all` does not exist in this repo's package.json
```suggestion
      command: yarn install --frozen-lockfile
```